recipes/pythesint/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "pythesint" %}
+{% set version = "1.2.0" %}
+{% set sha256 = "457ae24b8ffcee7a03ff9515c1578aec81d2bf3dd7a599d7cafc6e146e441b65" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
```

Comment:
If skipping with a selector, this can not be `noarch: python`:

http://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages

Reply:
Makes sense. Forgot to modify this. The plan is that we will make the package for all platforms and python versions, but need some fixes in the code for that.